pkgs/shells/marcel/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, dill
+, psutil
+, makeWrapper
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "marcel";
+  version = "0.17.0-unstable-2023-11-11";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "geophile";
+    repo = "marcel";
+    rev = "ed96ef5da31e7c099e22f11911af28eb4425d4d5";
+    hash = "sha256-503JqQb29Uwd3rm2r49accFl3pWjaX/BNziO4iZJmJ4=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    dill
+    psutil
+  ];
+
+  doCheck = false;
```

Comment:
please add a comment explaining why

Reply:
@KUD-00 commented in the thread in the marcel repo saying that the tests write to the HOME directory: https://github.com/geophile/marcel/issues/16#issuecomment-1810423637

Maybe this would allow them to run?
```nix
preCheck = ''
    export HOME=$(mktemp -d)
'';
```

Reply:
I tried `export HOME=$(mktemp -d)`, and it partially solved the problem, but there're some test like this:

```bash
ls -0 ~root | map (f: f.path) failed:
    expected:
<<<['/root']>>>
    actual:
<<<['/build']>>>
```

many tests based on hardcoded '/tmp' folder, and many of them use `sudo`, i think these won't work with nix sandbox environment.

@eclairevoyant  @arcuru What do you think? Or is it better to `doCheck = false`, I have already add a comment to this line.